staff/francis-hrom/strings/spell.test.js

```diff
@@ -0,0 +1,2 @@
+console.assert(spell("Hi")=="",'There is an error with input "Hi"');
```

Comment:
spell cannot be tested for now, as it does not return anything, it just sends characters to the console. so remove this test.